server/prisma/seed.ts

```diff
@@ -80,6 +80,155 @@ async function main() {
     },
   });
 
+  const classificationSystem = await prisma.classificationSystems.upsert({
+    where: { name: "Common Core" },
+    update: { name: "Common Core" },
+    create: { name: "Common Core" },
+  });
+  const { id: commonCoreId } =
```

Comment:
You don't need this second query, but can get the id directly from the `upsert`.

Reply:
Fixed.